app.js

```diff
@@ -1,12 +1,33 @@
 'use strict'
```

Comment:
For the next project, let's get your js files out of the root and into their own ``scripts`` folder.